pkg/apis/kudo/v1beta1/operator_types_helpers.go

```diff
@@ -0,0 +1,17 @@
+package v1beta1
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetOperator(namespacedName types.NamespacedName, c client.Client) (*Operator, error) {
```

Comment:
What do we gain from using `types.NamespacedName` here?
Our functions currently use separate arguments for object and namespace, by introducing `types.NamespacedName` here we end up with calling `v1beta1.GetOperator(types.NamespacedName{Name: name, Namespace: r.ns}, r.c)` instead of the much shorter `v1beta1.GetOperator(name, r.ns, rc)`.

Reply:
Ok.